DFC.App.DiscoverSkillsCareers.Services/Api/ApiService.cs

```diff
@@ -0,0 +1,128 @@
+        }
+
+        public async Task<bool> NewSession(string assessmentType)
+        {
+            var newSessionResponse = await assessmentApiService.NewSession(assessmentType).ConfigureAwait(false);
+            if (newSessionResponse != null)
+            {
+                sessionService.SetValue(SessionKey.SessionId, newSessionResponse.SessionId);
+            }
+
+            return newSessionResponse != null;
+        }
+
+        public async Task<GetQuestionResponse> GetQuestion(string assessmentType, int questionNumber)
+        {
+            Validate();
+
+            var getQuestionResponse = await assessmentApiService.GetQuestion(GetSessionId(), assessmentType, questionNumber).ConfigureAwait(false);
+
+            return getQuestionResponse;
+        }
+
+        public async Task<PostAnswerResponse> AnswerQuestion(string assessmentType, int questionNumber, string answer)
+        {
+            Validate();
+
+            var questionSetResponse = await GetQuestion(assessmentType, questionNumber).ConfigureAwait(false);
+
+            var questionIdFull = $"{questionSetResponse.QuestionSetVersion}-{questionNumber}";
+            var post = new PostAnswerRequest() { QuestionId = questionIdFull, SelectedOption = answer };
+            var answerQuestionResponse = await assessmentApiService.AnswerQuestion(GetSessionId(), post).ConfigureAwait(false);
+
+            return answerQuestionResponse;
+        }
+
+        public async Task<GetAssessmentResponse> GetAssessment()
+        {
+            Validate();
+
+            var response = await assessmentApiService.GetAssessment(GetSessionId()).ConfigureAwait(false);
+
+            return response;
+        }
+
+        public async Task<SendEmailResponse> SendEmail(string domain, string emailAddress, string templateId)
+        {
+            Validate();
+
+            var sendEmailResponse = await assessmentApiService.SendEmail(GetSessionId(), domain, emailAddress, templateId).ConfigureAwait(false);
+
+            return sendEmailResponse;
+        }
+
+        public async Task<GetResultsResponse> GetResults()
+        {
+            Validate();
+
+            return await resultsApiService.GetResults(GetSessionId()).ConfigureAwait(false);
+        }
+
+        public async Task<FilterAssessmentResponse> FilterAssessment(string jobCategory)
+        {
+            Validate();
+
+            return await assessmentApiService.FilterAssessment(GetSessionId(), jobCategory).ConfigureAwait(false);
+        }
+
+        public async Task<string> Reload(string referenceCode)
+        {
+            var sessionId = sessionIdToCodeConverter.GetSessionId(referenceCode);
+
+            var assessment = await assessmentApiService.GetAssessment(sessionId).ConfigureAwait(false);
+
+            sessionService.SetValue(SessionKey.SessionId, assessment.SessionId);
+
+            return assessment.SessionId;
+        }
+
+        private void Validate()
+        {
+            ValidateSession();
```

Comment:
Why have an obsolete method.  As your not doing any logic, did you need the original calling method.  Couldn't you have just called validatesession from the methods?